.travis.yml

```diff
@@ -14,8 +13,27 @@ os:
 cache:
   directories:
     - "$HOME/.cargo"
+    - target
 before_cache:
-  - rm -rf "$HOME/.cargo/registry"
+  - |
+    cargo install \
+      --debug \
+      --force \
+      --git https://github.com/matthiaskrgr/cargo-cache \
```

Comment:
Can we use the crates.io version?

Reply:
It is not released yet.

Reply:
Oh, and the reason I don't use crates.io version is that it is slow to compile.
The git version has features "ci_autoclean" which cuts down compile time to
seconds.